cdap-security/src/main/java/co/cask/cdap/security/server/ExternalAuthenticationServer.java

```diff
@@ -127,91 +127,87 @@ public InetSocketAddress getSocketAddress() {
 
   @Override
   protected void startUp() throws Exception {
+    server = new Server();
+
     try {
-      server = new Server();
+      bindAddress = InetAddress.getByName(cConfiguration.get(Constants.Security.AUTH_SERVER_BIND_ADDRESS));
+    } catch (UnknownHostException e) {
+      LOG.error("Error finding host to connect to.", e);
```

Comment:
Don't log if you are rethrowing an exception.

Reply:
ah, this was previous code. Will fix it.